### DISCUSSION TITLE
renovate merging PRs with failing tests

### DISCUSSION THREAD

**Discussion by mjfwebb** [renovatebot/renovate#26496] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

For example, on https://github.com/neo4j/graphql/pull/4463 the PR was merged even when tests were failing and the quality gate was skipped.

### Relevant debug logs

It's unclear which log to provide because there's no reference to the job id in the dashboard and there are many renovate jobs running all the time.

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by viceice** [comment]
renovate switched to platform automerge since a while, so GitHub is doing the merge.

you need to disable platform automerge or add branch protection with mandatory checks.
